java/rocksjni/portal.h

```diff
@@ -5283,6 +5279,14 @@ class TickerTypeJni {
         return -0x59;
       case ROCKSDB_NAMESPACE::Tickers::ICE_FILE_READ_COUNT:
         return -0x5A;
+      case ROCKSDB_NAMESPACE::Tickers::COOL_FILE_READ_BYTES:
+        return -0x5B;
+      case ROCKSDB_NAMESPACE::Tickers::COOL_FILE_READ_COUNT:
+        return -0x5C;
+      case ROCKSDB_NAMESPACE::Tickers::NUMBER_WBWI_INGEST:
+        return -0x5D;
+      case ROCKSDB_NAMESPACE::Tickers::SST_USER_DEFINED_INDEX_LOAD_FAIL_COUNT:
+        return -0x5E;
```

Comment:
I don't see these in java/src/main/java/org/rocksdb/TickerType.java

Reply:
Thanks. AI is bad at this stuff